Drop-In/src/main/java/com/braintreepayments/api/SupportedPaymentMethodsFragment.java

```diff
@@ -201,7 +199,7 @@ private void showVaultedPaymentMethods() {
         }
         
         if (paymentMethodNonces != null && paymentMethodNonces.size() > 0) {
-            supportedPaymentMethodsHeader.setText(R.string.bt_other);
+            supportedPaymentMethodsHeader.setVisibility(View.VISIBLE);
```

Comment:
We updated the supported payment methods header to start with text "Other" and be invisible if no vaulted payment methods exist - this is because there is now a header that says "Select Payment Method" that is always shown, so we only need the "Other" section header when vaulted payment methods exist. See screenshots in PR description.